falcon/routing/converters.py

```diff
@@ -59,6 +65,28 @@ def convert(self, fragment):
         return value
 
 
+class DateTimeConverter(object):
+    """Converts a field value to a datetime.
+
+    Keyword Args:
+        format_string (str): String used to parse the param value
+            into a datetime. Any format recognized by strptime() is
+            supported (default ``'%Y-%m-%dT%H:%M:%SZ'``).
+    """
+
+    __slots__ = ('_format_string', '_strptime')
+
+    def __init__(self, format_string='%Y-%m-%dT%H:%M:%SZ'):
+        self._format_string = format_string
+
+    def convert(self, fragment):
+        try:
+            return strptime(fragment, self._format_string)
+        except ValueError:
+            return None
+
+
 BUILTIN = (
     ('int', IntConverter),
+    ('dt', DateTimeConverter),
```

Comment:
There is no explicit objection to having this converter but what is the purpose of adding this? And where did the need come from? Only asking to understand requirements

Even though you provided format_string as a keyword argument, a user that may need to provide a different format will have to create a customer class (maybe by inheriting from DateTimeConverter) to provide a format of their choice. Is this ok or expected for usage?